src/components/hooks/useDarkMode.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect } from 'react'
+import { useLocalStorage } from "./useLocalStorage"
+
+export const useDarkMode = () => {
+	const [darkMode, setDarkMode] = useLocalStorage("dark-theme");
```

Comment:
From the definition of `useLocalStorage` it expects two parameters, but here only one param is passed, any reason?

Reply:
Merge conflicts have been fixed